src/calendar-list/index.js

```diff
@@ -130,8 +132,15 @@ class CalendarList extends Component {
           break;
         }
       }
+      if (scrollAmount < 0) {
+        this.setState({addMarginTop: Math.abs(scrollAmount)});
```

Comment:
You're invoking render while scrolling the list... Isn't there a better solution?